src/style.css

```diff
@@ -1,3 +1,75 @@
+html {
+  --main-color: #000;
+  --link-color: #00acdc;
```

Comment:
I would suggest to make variable name more informative, like active-color, moyo-blue, link-active-color, etc